scripts/flashing/silabs_firmware_utils.py

```diff
@@ -169,6 +171,45 @@ def actions(self):
 
         return self
 
+    def _platform_wrapper_args(self, args):
+        """Called from make_wrapper() to optionally manipulate arguments."""
+        # Generate the flashbundle.txt file and copy the firmware utils.
+        if args.flashbundle_file is not None:
+            # Generate the flashbundle contents.
+            # Copy the platform-specific and general firmware utils to the same directory as the wrapper.
+            flashbundle_contents = os.path.basename(args.output)
+            if args.application is not None:
+                flashbundle_contents += "\n" + os.path.basename(args.application)
+            output_dir = os.path.dirname(args.output) or "."
+            if args.platform_firmware_utils is not None:
+                flashbundle_contents += "\n" + os.path.basename(args.platform_firmware_utils)
+                shutil.copy(args.platform_firmware_utils, output_dir)
+            if args.firmware_utils is not None:
+                flashbundle_contents += "\n" + os.path.basename(args.firmware_utils)
+                shutil.copy(args.firmware_utils, output_dir)
+
+            # Create the flashbundle file.
+            try:
+                with open(args.flashbundle_file, 'w') as flashbundle_file:
+                    flashbundle_file.write(flashbundle_contents.strip())
+            except OSError as exception:
+                print(exception, sys.stderr)
```

Comment:
could you add a comment as to why we print this instead of propagating?

Would https://docs.python.org/3/library/traceback.html#traceback.print_last result in a nicer printout maybe?

Reply:
Good catch.  Instead of printing here, this needs to propagate back to `maker_wrapper` so it knows to `return 1` instead of just moving on.  I also updated `make_wrapper` to use `traceback.print_last()` instead of `print(exception, sys.stderr)`.